Sources/ApolloCodegenLib/Templates/RenderingHelpers/OperationTemplateRenderer.swift

```diff
@@ -50,4 +50,40 @@ extension OperationTemplateRenderer {
       """
   }
 
+  func DeferredProperties(
```

Comment:
The naming `DeferredProperties` confused me here. Can we just rename this to `HasDeferredFragmentsTemplate`